bin/test_travis.sh

```diff
@@ -92,3 +92,5 @@ if [[ "${TEST_CPP}" != "no" ]]; then
     ./a.out
 fi
 
+echo "Checking whether all header files are installed:"
+python $SOURCE_DIR/bin/test_make_install.py $our_install_dir/include/symengine/ $SOURCE_DIR/symengine|| exit 1
```

Comment:
Is the `|| exit 1` necessary? I would think it should work without it as well, since the Python script should return 0 or 1 anyway.